uaclient/tests/test_cli_status.py

```diff
@@ -492,7 +492,7 @@ def test_attached_formats(
                 "created_at": account_created_at,
                 "external_account_ids": [{"IDs": ["id1"], "Origin": "AWS"}],
             },
-            "config_path": "config_path",
+            "config_path": None,
```

Comment:
Question: is there any test to cover when the config path would not be `None` ?

Reply:
Currently not, because of the way we are using the `FakeConfig` fixture, which creates a `UAConfig` object with a config dict ready. Maybe we should better address this in the integration tests

Reply:
Good catch, I think it'd be sufficient to just add a quick unit test for `get_config_path`